src/Workspaces/Core/Portable/Remote/WellKnownSynchronizationKind.cs

```diff
@@ -14,7 +14,6 @@ internal enum WellKnownSynchronizationKind
     // Solution snapshot state, only referencing actual user (non-generated) documents, options, and references.
     SolutionState,
     ProjectState,
-    DocumentState,
```

Comment:
this is no longer synced to/from oop.  instead, we just inline it's two checkums (attributes and text checksums) to the projectstatechecksum.

Reply:
this cuts the total number of calls form oop to the host in a 3rd.